Experiments/Experiments.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 53;
+	objectVersion = 54;
```

Comment:
This `objectVersion` was added automatically when updating the target, and seems to indicate the Xcode compatibility version, which would require a minimum of Xcode 12 to load the project.